pkg/models/host.go

```diff
@@ -148,6 +164,19 @@ func (h *Host) Download(remoteFile string, localFile string, timeout time.Durati
 	return err
 }
 
+// ReadFileBytes downloads a file from the remote server to a byte array
+func (h *Host) ReadFileBytes(remoteFile string, timeout time.Duration) ([]byte, error) {
```

Comment:
ohh I see https://github.com/melbahja/goph/blob/master/client.go does not support uploading or downloading bytes. such a shame. maybe we can propose a PR to goph.

Reply:
agree